src/v/redpanda/admin/security.cc

```diff
@@ -883,7 +884,33 @@ ss::future<std::unique_ptr<ss::http::reply>> admin_server::delete_role_handler(
           ss::http::reply::status_type::no_content,
           ss::json::json_void{});
     }
+
     co_await throw_on_error(*req, err, model::controller_ntp);
+
+    if (delete_acls) {
+        security::acl_binding_filter role_binding_filter{
+          security::resource_pattern_filter::any(),
+          security::acl_entry_filter{
+            security::role::to_principal(role_name()),
+            std::nullopt,
+            std::nullopt,
+            std::nullopt}};
+
+        auto results
+          = co_await _controller->get_security_frontend().local().delete_acls(
+            {std::move(role_binding_filter)}, 5s);
+
+        for (const auto& r : results) {
+            co_await throw_on_error(*req, r.error, model::controller_ntp);
```

Comment:
Might be preferable to fail silently here. Idk. Any opinions?

Reply:
I'd probably log it... or maybe include it in the response but still reutrn a 2XX response code?